src/rom_header_rhh.c

```diff
@@ -0,0 +1,26 @@
+#include "global.h"
+#include "constants/expansion.h"
+
+// Similar to the GF ROM header, this struct allows external programs to
+// detect details about Expansion.
+// For this structure to be useful we have to maintain backwards binary
+// compatibility. This means that we should only ever append data to the
+// end. If there are any structs as members then those structs should
+// not be modified after being introduced.
+struct RHHRomHeader
+{
+    char rhh_magic[4]; // 'PORY'. Useful to locate the header if it shifts.
+    u8 expansionVersionMajor;
+    u8 expansionVersionMinor;
+    u8 expansionVersionPatch;
+    bool8 expansionVersionTagged;
```

Comment:
```suggestion
    bool8 expansionVersionTagged:1;
    u8 padding:7;
```

Reply:
What's the reasoning behind it actually?

Reply:
Behind `expansionVersionTagged`? or the padding?

Reply:
the padding

Reply:
I figured that since we would only need a single bit for this, might as well just use what we need?